src/Symfony/Component/Form/Extension/Core/Type/DateIntervalType.php

```diff
@@ -135,7 +129,11 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 }
             }
             if ($options['with_invert']) {
-                $builder->add('invert', 'Symfony\Component\Form\Extension\Core\Type\CheckboxType', $invertOptions);
+                $builder->add('invert', 'Symfony\Component\Form\Extension\Core\Type\CheckboxType', array(
+                    'error_bubbling' => true,
+                    'required' => false,
+                    'translation_domain' => $options['translation_domain'],
```

Comment:
Anyway it should be kept IMHO. It's used to translate the `Invert` label, so it makes sense to use the same translation domain as for the whole widget.